packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -317,6 +318,79 @@ curl -v -X POST \
 ##### Response Example
 
 
+```json
+{
+  "id": "00ub0oNGTSWTBKOLGLNR",
+  "status": "ACTIVE",
+  "created": "2013-07-02T21:36:25.344Z",
+  "activated": null,
+  "statusChanged": null,
+  "lastLogin": null,
+  "lastUpdated": "2013-07-02T21:36:25.344Z",
+  "passwordChanged": "2013-07-02T21:36:25.344Z",
+  "profile": {
+    "firstName": "Isaac",
+    "lastName": "Brock",
+    "email": "isaac.brock@example.com",
+    "login": "isaac.brock@example.com",
+    "mobilePhone": "555-415-1337"
+  },
+  "credentials": {
+    "password": {},
+    "provider": {
+      "type": "IMPORT",
+      "name": "IMPORT"
+    }
+  },
+  "_links": {
+    "activate": {
+      "href": "https://${yourOktaDomain}/api/v1/users/00ub0oNGTSWTBKOLGLNR/lifecycle/activate"
+    },
+    "self": {
+      "href": "https://${yourOktaDomain}/api/v1/users/00ub0oNGTSWTBKOLGLNR"
+    }
+  }
+}
+```
+
+#### Create User with Password Import Inline Hook
+
+
+Creates a user with a specified [password hook](#password-hook-object). See [Password Import Inline Hook](/docs/reference/password-hook/) for more details.
```

Comment:
'Creates a user with a specified [password hook](#password-hook-object)' -- At this point, the object doesn't actually specify a particular hook to use, it just specifies that the single possible password hook (default) should be used. So, might be clearer to say `Creates a user with the [Password Hook](#password-hook-object) object specifying that a Password Inline Hook should be used to handle password verification.` (Also, should capitalize the name of the object.)

Reply:
Done!